mkdocs/docs/encodings.md

```diff
@@ -2,153 +2,138 @@
+For implementation details, refer to [Protobuf](https://protobuf.dev/programming-guides/encoding/#varints).
 
 #### SIMD-FastPFOR
 
-Bit-aligned null suppression technique that compresses an integer using a minimal number of bits
-by using a patched approach to store exceptions (outliers) in a separate section to keep the overall bit width small.
+A bit-aligned null suppression technique that compresses integers using a minimal number of bits.
+Uses a patched approach to store exceptions (outliers) separately, keeping the overall bit width small.
 
-see [https://arxiv.org/pdf/1209.2137.pdf](https://arxiv.org/pdf/1209.2137.pdf) and [https://ayende.com/blog/199524-C/integer-compression-the-fastpfor-code](https://ayende.com/blog/199524-C/integer-compression-the-fastpfor-code)
+Refer to:
+- [https://arxiv.org/pdf/1209.2137.pdf](https://arxiv.org/pdf/1209.2137.pdf)
+- [https://ayende.com/blog/199524-C/integer-compression-the-fastpfor-code](https://ayende.com/blog/199524-C/integer-compression-the-fastpfor-code)
 
-Available implementations:
+**Available implementations**:
 
-- C++: [https://github.com/lemire/FastPFor](https://github.com/lemire/FastPFor)
-- Java: [https://github.com/lemire/JavaFastPFOR](https://github.com/lemire/JavaFastPFOR)
-- C#: [https://github.com/Genbox/CSharpFastPFOR](https://github.com/Genbox/CSharpFastPFOR)
-- Js/WebAssembly: Work in Progress (higher implementation complexity)
+- **C++**: [https://github.com/fast-pack/FastPFOR](https://github.com/fast-pack/FastPFOR)
+- **Java**: [https://github.com/fast-pack/JavaFastPFOR](https://github.com/fast-pack/JavaFastPFOR)
+- **C#**: [https://github.com/Genbox/CSharpFastPFOR](https://github.com/Genbox/CSharpFastPFOR)
+- **JS/WebAssembly**: Work in progress (higher implementation complexity)
 
-## Float encodings
+## Float Encodings
 
-### Adaptive Lossless floating-Point Compression (ALP)
+### Adaptive Lossless Floating-Point Compression (ALP)
 
-See [https://dl.acm.org/doi/pdf/10.1145/3626717](https://dl.acm.org/doi/pdf/10.1145/3626717) for a detailed explanation.
-ALP is a lossless floating-point compression technique that is used to compress the float values in MLT.
+A lossless floating-point compression technique used in MLT.
+For a detailed explanation, refer to [this paper](https://dl.acm.org/doi/pdf/10.1145/3626717).
 
-Available implementations:
+**Available implementations**:
 
-- C++: [https://github.com/duckdb/duckdb/tree/16830ded9b4882b90f35c2d7e2d740547ae7d3fc/src/include/duckdb/storage/compression/alp](https://github.com/duckdb/duckdb/tree/16830ded9b4882b90f35c2d7e2d740547ae7d3fc/src/include/duckdb/storage/compression/alp)
-- Java: Work in Progress
-- JS/WebAssembly: Work in Progress
+- **C++**: [https://github.com/duckdb/duckdb/tree/16830ded9b4882b90f35c2d7e2d740547ae7d3fc/src/include/duckdb/storage/compression/alp](https://github.com/duckdb/duckdb/tree/16830ded9b4882b90f35c2d7e2d740547ae7d3fc/src/include/duckdb/storage/compression/alp)
```

Comment:
We do this pattern a few times in the article.
I think we should do the following instead in those cases


```suggestion
- **C++**: <https://github.com/duckdb/duckdb/tree/16830ded9b4882b90f35c2d7e2d740547ae7d3fc/src/include/duckdb/storage/compression/alp>
```

Reply:
thx, fixed